src/rtctools_heat_network/workflows/utils/adapt_profiles.py

```diff
@@ -0,0 +1,105 @@
+import datetime
+import logging
+
+import numpy as np
+
+from rtctools.data.storage import DataStore
+
+logger = logging.getLogger("WarmingUP-MPC")
+logger.setLevel(logging.INFO)
+
+
+def adapt_hourly_profile_to_common(problem, problem_day_steps):
```

Comment:
I want the function name to be slightly more descriptive. Can we change the name of this function to: adapt_hourly_year_profile_to_day_averaged_with_hourly_peak_day()

Reply:
Agree

Reply:
Fixed